.github/workflows/deploy-examples.yml

```diff
@@ -63,3 +64,4 @@ jobs:
           OXYGEN_DEPLOYMENT_TOKEN_1000014928: ${{ secrets.OXYGEN_DEPLOYMENT_TOKEN_1000014928 }}
           OXYGEN_DEPLOYMENT_TOKEN_1000014929: ${{ secrets.OXYGEN_DEPLOYMENT_TOKEN_1000014929 }}
           OXYGEN_DEPLOYMENT_TOKEN_1000014930: ${{ secrets.OXYGEN_DEPLOYMENT_TOKEN_1000014930 }}
+          OXYGEN_DEPLOYMENT_TOKEN_1000019891: ${{ secrets.OXYGEN_DEPLOYMENT_TOKEN_1000019891 }}
```

Comment:
We should add the other examples as well since they are all diffs now. Do we need to create a new storefront manually for each of them to get this token?

Reply:
Yeah. It's a bit of a pain.